src/megachatapi.h

```diff
@@ -2056,6 +2056,25 @@ class MegaChatApi
      */
     MegaChatListItemList *getChatListItems();
 
+    /**
+     * @brief Get all chatrooms (1on1 and groupal) that contains a certain set of participants
+     *
+     * It is needed to have successfully called \c MegaChatApi::init (the initialization
+     * state should be \c MegaChatApi::INIT_OFFLINE_SESSION or \c MegaChatApi::INIT_ONLINE_SESSION)
+     * before calling this function.
+     *
+     * Note that MegaChatListItem objects don't include as much information as
+     * MegaChatRoom objects, but a limited set of data that is usually displayed
+     * at the list of chatrooms, like the title of the chat or the unread count.
+     *
+     * This function returns even archived chatrooms.
+     *
+     * You take the ownership of the returned value
+     *
```

Comment:
I miss the doc for the `@param peers`. Please, remark the list should not include your own user, since MEGAchat doesn't include them in the map of members for each chatroom.